src/components/Table/TableHead/TableHead.jsx

```diff
@@ -323,13 +327,15 @@ const TableHead = ({
       )}
       {activeBar === 'column' && (
         <ColumnHeaderRow
+          erRow
```

Comment:
is this a typo?

Reply:
Yes, must have been a burglar. It is a thing, they break into people's homes and add typos everywhere.